core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -840,6 +849,42 @@ class BrokerDynamicThreadPool(server: KafkaBroker) extends BrokerReconfigurable
   }
 }
 
+
+class RemoteLogDynamicThreadPool(server: KafkaBroker) extends BrokerReconfigurable {
+
+  override def reconfigurableConfigs: Set[String] = Set(
+    RemoteLogManagerConfig.REMOTE_LOG_MANAGER_COPIER_THREAD_POOL_SIZE_PROP,
+    RemoteLogManagerConfig.REMOTE_LOG_MANAGER_EXPIRATION_THREAD_POOL_SIZE_PROP,
+    RemoteLogManagerConfig.REMOTE_LOG_READER_THREADS_PROP)
+
+  override def validateReconfiguration(newConfig: KafkaConfig): Unit = {
+    DynamicThreadPool.validateReconfiguration(server.config, newConfig)
+  }
+
+  override def reconfigure(oldConfig: KafkaConfig, newConfig: KafkaConfig): Unit = {
+    if (server.remoteLogManagerOpt.isEmpty) {
+      return
+    }
+
+    val remoteLogManager = server.remoteLogManagerOpt
```

Comment:
We can either rename the variable (or) get the value from the option: 

```
val remoteLogManagerOpt = server.remoteLogManagerOpt
```

(or)

```
val remoteLogManager = server.remoteLogManagerOpt.get
```